tsidp-server.go

```diff
@@ -148,23 +154,26 @@ func main() {
 			slog.Error("failed to get local client", slog.Any("error", err))
 			os.Exit(1)
 		}
-		var ln net.Listener
-		if *flagFunnel {
-			if err := ipn.CheckFunnelAccess(uint16(*flagPort), st.Self); err != nil {
-				slog.Error("funnel access denied", slog.Any("error", err))
+
+		if !*flagDisableTCP {
```

Comment:
As it stands, I don't really see much of a use-case for `--unix-socket` without `--use-local-tailscaled`. The nginx server will never be able to listen on the same domain as `tsnet` and there will _have_ to be a local `tailscaled` session for ingress through nginx to be tagged correctly.

I added the disable flag here mostly for completeness.